docs/docs/marker.mdx

```diff
@@ -0,0 +1,280 @@
+1309ed25d6fa45beb74d0862d37289a5,0,marker_mood_expressed,median(number of preceding user turns),1.0
+2cf4106e76a041dfb2d4f75bd6ae804b,0,marker_mood_expressed,median(number of preceding user turns),1.0
+79e7604d2acc442d9f78e00b173786c9,0,marker_mood_expressed,median(number of preceding user turns),nan
+1309ed25d6fa45beb74d0862d37289a5,0,marker_mood_expressed,min(number of preceding user turns),1
+2cf4106e76a041dfb2d4f75bd6ae804b,0,marker_mood_expressed,min(number of preceding user turns),1
+79e7604d2acc442d9f78e00b173786c9,0,marker_mood_expressed,min(number of preceding user turns),nan
+:
+```
+
+stats-per-session.csv
+
+```
+sender_id,session_idx,marker,statistic,value
+all,nan,-,total_number_of_sessions,3
+all,nan,marker_cheer_up,number_of_sessions_where_marker_applies_at_least_once,0
+all,nan,marker_cheer_up,percentage_of_sessions_where_marker_applies_at_least_once,0.0
+all,nan,marker_mood_expressed,number_of_sessions_where_marker_applies_at_least_once,2
+all,nan,marker_mood_expressed,percentage_of_sessions_where_marker_applies_at_least_once,66.667
+all,nan,marker_mood_expressed_and_name_provided,number_of_sessions_where_marker_applies_at_least_once,0
+all,nan,marker_mood_expressed_and_name_provided,percentage_of_sessions_where_marker_applies_at_least_once,0.0
+:
+```
+
+stats-overall.csv
+
+## Configuring the CLI command
+
+You can configure the marker extraction and statistics computation using the following arguments:
+
+```
+usage: rasa evaluate markers [-h] [-v] [-vv] [--quiet] [--config CONFIG]
+                             [--no-stats | --stats-file-prefix [STATS_FILE_PREFIX]]
+                             [--endpoints ENDPOINTS] [-d DOMAIN]
+                             output_filename {first_n,sample,all} ...
+
+positional arguments:
+  output_filename       The filename to write the extracted markers to (CSV format).
+  {first_n,sample,all}
+    first_n             Select trackers sequentially until N are taken.
+    sample              Select trackers by sampling N.
+    all                 Select all trackers.
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --config CONFIG       The config file(s) containing marker definitions. This can be a single YAML
+                        file, or a directory that contains several files with marker definitions in
+                        it. The content of these files will be read and merged together. (default:
+                        markers.yml)
+  --no-stats            Do not compute summary statistics. (default: True)
+  --stats-file-prefix [STATS_FILE_PREFIX]
+                        The common file prefix of the files where we write out the compute
+                        statistics. More precisely, the file prefix must consist of a common path
+                        plus a common file prefix, to which suffixes `-overall.csv` and `-per-
+                        session.csv` will be added automatically. (default: stats)
+  --endpoints ENDPOINTS
```

Comment:
I think it's probably worth mentioning these other configuration files (endpoints config, and domain) that the tool is looking for further up in the body of the text as well - would be good for users to know about the domain validation behaviour as well as the fact it needs the endpoints config file to actually connect to a tracker store and won't work without one (and it will assume default paths if none are specified)